### PR TITLE
Added AWS peering script

### PR DESCRIPTION
**Pull request by lewismarshall**
Created a script to create peered VPC's and default routes on both VPC's.

Will take three parameters:

```
vpc_cidr=$1
remote_profile=$2
remote_cidr=$3
```

It will require the remote VPC to be accessible using an aws profile...
Tested as part of https://gitlab.digital.homeoffice.gov.uk/Devops/hod-ci/blob/master/scripts/peer_vpc.sh.

### CODE REVIEW

**Comment by lewismarshall** [comment]
@vaijab @jon-shanks @gambol99 FYI...

File: bin/peer_vpc.sh [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+if [[ -n $(get_route_entry ${remote_vpc_id} ${remote_peering_id} ${remote_profile}) ]]; then
+  echo "Remote Route exists already"
+else
+  echo "--- Creating remote route in ${remote_vpc_id} to ${vpc_cidr}..."
+  aws ec2 \
+    --profile ${remote_profile} \
+    create-route \
+    --route-table-id ${remote_route_table_id} \
+    --destination-cidr-block ${vpc_cidr} --gateway-id ${remote_peering_id} > /dev/null
+fi
```
Comment (vaijab) [inline bin/peer_vpc.sh:147]: No end of line :-D

**Comment by vaijab** [comment]
I'd suggest adding some sort of help or comments in the script, because you just mentioned how to use it in the PR.

**Comment by lewismarshall** [comment]
Added help in the script...

**Comment by vaijab** [comment]
LGTM

**Comment by vaijab** [comment]
Good to merge @lewismarshall ?

**Comment by lewismarshall** [comment]
Nope, not just yet... going to get correct route table

On Thursday, 28 January 2016, Vaidas Jablonskis notifications@github.com
wrote:

> Good to merge @lewismarshall https://github.com/lewismarshall ?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/UKHomeOffice/aws-dsp-toolset/pull/9#issuecomment-176108773
> .

## 

Lewis

Mobile: 0788 757 3297

My web sites:

blog.deploymentsource.com
http://blog.deploymentsource.com/
uk.linkedin.com/in/lewismarshall
http://uk.linkedin.com/in/lewismarshallwww.itkeys.co.uk

**Comment by lewismarshall** [comment]
All good to go - added option to use tags to find routetables (not required if "main" routetable unused).

**Comment by lewismarshall** [comment]
Be good to get this in a version of toolset so I can delete from the dsp-ci repo. Routes all tested with docker worker...

**Comment by vaijab** [comment]
LGTM

**Pull request merged**